app/code/Magento/CompareListGraphQl/Model/Service/Collection/ComparableItems.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\CompareListGraphQl\Model\Service\Collection;
+
+use Magento\Catalog\Helper\Product\Compare;
+use Magento\Catalog\Model\Config as CatalogConfig;
+use Magento\Catalog\Model\Product\Visibility as CatalogProductVisibility;
+use Magento\Catalog\Model\ResourceModel\Product\Compare\Item\Collection;
+use Magento\Catalog\Model\ResourceModel\Product\Compare\Item\CollectionFactory as CompareItemsCollectionFactory;
+use Magento\Framework\GraphQl\Query\Resolver\ContextInterface;
+
+/**
+ * Get collection with comparable items
+ */
+class ComparableItems
```

Comment:
Please, make sure that the service name reflects its purpose. It's recommended to use a verb in the very beginning of the service like `GetComparableItemsCollection`. Here as well, you may use `execute` method as the entry point. 

This recommendation apply to all other services.

Thank you

Reply:
**Done**